mkdocs.yml

```diff
@@ -123,10 +125,15 @@ nav:
     - Enviorment Variables:     reference/enviorment.md
     - Shift Python API:         reference/api.md
   - Integrations & Resources:
-    - Maya:                     integration_resources/maya.md
-    - Houdini:                  integration_resources/houdini.md
-    - Nuke:                     integration_resources/nuke.md
-    - USD:                      integration_resources/usd.md
-    - Deadline:                 integration_resources/deadline.md
+    - integration_resources/integrations_resources.md
+    - Software Platforms:
+      - Maya:                     integration_resources/software/maya.md
+      - Houdini:                  integration_resources/software/houdini.md
+      - Nuke:                     integration_resources/software/nuke.md
+      - Resolve:                  integration_resources/software/resolve.md
+      - Deadline:                 integration_resources/software/deadline.md
+    - Open Resources:
```

Comment:
Can we change the name for this section to: "Frameworks Catalogs Resources"?
What do you think @Inbibo/shift ?

Reply:
I tried to avoid using Catalogs because some repos will also include Plugins. It could be Frameworks Resources

Reply:
Or maybe `Open Catalog & Plugin Resources`?

Reply:
I like the concept of `Framework Resources` as Julia said, also `External Resources` sounds good for me. 
If necessary we can add at the beginning of the usd and ftrack documentation pages what means to be inside that parent section, to make that more comprehensive.

Reply:
Why not "simply" `Other Resources` ?
This would open the door open for us to define the chapter as we need.
Then once the content clearly defines over time, we can change the name to better fit its purpose?

Reply:
I am up for @mromeoinbibo solution

Reply:
Then @juliavisserinbibo we go with `Other Resources`.

Reply:
Done